integreat_cms/cms/templates/contacts/contact_list_row.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n %}
+<tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
+    <td class="pl-2">
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.title }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.name }}
+        </a>
+    </td>
+    <td>
+        <a href="" class="block py-3 px-2 text-gray-800">
+            {{ contact.poi }}
+        </a>
```

Comment:
it probably helps users if we add here link to the referenced POI?

Reply:
Alright. Sounds reason-able. I will style it like a link to make it obvious :)